helix-core/src/test/java/org/apache/helix/integration/common/ZkStandAloneCMTestBase.java

```diff
@@ -28,8 +28,6 @@
 import org.apache.helix.integration.manager.MockParticipantManager;
 import org.apache.helix.tools.ClusterVerifiers.BestPossibleExternalViewVerifier;
 import org.apache.helix.tools.ClusterVerifiers.ZkHelixClusterVerifier;
-import org.slf4j.Logger;
```

Comment:
Good that you are removing redundant imports, but let us have a separate PR for this type of changes. Fine for this PR.

Reply:
Sorry I guess these are just auto-done by IDE